cli/cmd/plugin/ui/web/tanzu-ui/src/views/temp/progress.component.tsx

```diff
@@ -0,0 +1,111 @@
+// React imports
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+
+// Library imports
+import { CdsButton } from '@cds/react/button';
+import { LazyLog } from 'react-lazylog';
+import styled from 'styled-components';
+
+// App imports
+import { useWebsocketService, WebsocketService, WsOperations } from '../../shared/services/Websocket.service';
+import './progress.component.scss';
+
+const LogViewContainer = styled.section`
+    height: 500px;
+    width: 900px;
+`;
+
+export const LogTypes = {
```

Comment:
If we use this logTypes all over the places, I think we should move it to App.constants.tsx or create another constant file for it.

Reply:
Yes, I was thinking that we may want to create a component that wraps the LazyLog library, so we can reuse most of what I have done here in other 'deploy progress' pages. For example we will want to show deployment progress for management clusters, workload clusters, and unmanaged clusters. If we create a reusable component, then I think we would define the LogTypes in that reusable component. I do not think we will use LogTypes in other parts of the app. Does that sound good?

Reply:
Yes. It makes sense. Create a new wrapper component for the LayLog is a good idea.